components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/utils/RestApiPublisherUtils.java

```diff
@@ -346,4 +349,16 @@ public static String getSOAPOperation() {
                 + "\"required\":false,\"in\":\"header\"}],\"responses\":{\"200\":{\"description\":\"OK\"}}," +
                 "\"security\":[{\"default\":[]}],\"consumes\":[\"text/xml\",\"application/soap+xml\"]}}}";
     }
+
+    /**
+     * This method retrieves the Swagger Definition for an API to be displayed
+     * @param api API
```

Comment:
Missing apiProvider in the java docs

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/commit/eb9b7181c0f6cb5b9624602d91d28a7479bf551b